### DISCUSSION TITLE
AL ?

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#318] [Show and tell]
https://mfe01.tring.al/delta/106/out/u/1111_3.m3u8 seems like an internal Tring channel for marketers.

**Comment by deniscerri** [comment]
https://mfe01.tring.al/delta/105/out/u/618_13
https://mfe01.tring.al/delta/106/out/u/1114_3.m3u8
https://mfe01.tring.al/delta/106/out/u/1117_3.m3u8

Nr's smaller and greater don't work so it looks like its random

**Reply by BellezaEmporium** [reply]
As long as it doesn't change over time it's alright, the Marketing stream seems to have stopped but the others listed in al_tring.m3u do work.
